src/Database/Type/DateTimeType.php

```diff
@@ -188,6 +195,24 @@ public function setDatabaseTimezone($timezone)
         return $this;
     }
 
+    /**
+     * Set local timezone.
+     *
+     * This is the time zone used when marshalling strings to DateTime instances.
+     *
+     * @param string|\DateTimeZone|null $timezone Database timezone.
+     * @return $this
+     */
+    public function setLocalTimezone($timezone)
```

Comment:
This might be a confusing name since it is only used for marshalling and not converting from db to php type.

I'm not sure it's the wrong name though.

Reply:
Could this have 'user' in the name? That might help better indicate where/how it is meant to be used. I also wonder if we should have an example in the docs of how to build a middleware that uses this method to ensure requests always have their dates/times localized.

Reply:
I think user isn't distinct enough from server + local for many people including international users. We would have to be explicit like `setUserInterfaceTimezone()` or `setWebBrowserTimezone()` maybe.

Reply:
@markstory Yes, perhaps `setUserTimezone()`?
@othercorey I don't like the alternatives you have suggested :slightly_smiling_face:.

Reply:
`setUserTimezone()` is ok with me. It's documentable and probably covers more situations.

Reply:
`setUserTimezone()` works for me.